packages/core/src/components/Table/Body/Body.tsx

```diff
@@ -12,6 +12,16 @@ const Body: React.FC<Props> = React.memo(props => {
             TableComponentsCommonPropsContext
         ),
         { selectedRowIds, onRowSelection, onGroupedRowSelection, setUniqueIds, ...restProps } = props,
+        /* since minimap is positioned sticky with respect to the tbody, tbody should have full table width otherwise minimap positioning fails */
+        getNoResultRowColumnTemplate = useCallback(() => {
+            let scrollWidth = 0,
```

Comment:
You can do something like this ---> `scrollWidth = tableRef.current?.scrollWidth ?? 0;`